site/js-untar.d.ts

```diff
@@ -1,14 +1,21 @@
 declare module "js-untar" {
```

Comment:
Can we remove `js-untar` since we have the TarReader now?

Reply:
Yes, we can. I would like to make it in a diff PR tho. Idk what surprises it could cause.

Reply:
Makes sense!